bundles/org.eclipse.equinox.log.stream/META-INF/MANIFEST.MF

```diff
@@ -3,16 +3,13 @@ Bundle-ManifestVersion: 2
 Bundle-Name: %bundleName
 Bundle-Vendor: %bundleVendor
 Bundle-SymbolicName: org.eclipse.equinox.log.stream
-Bundle-Version: 1.0.400.qualifier
+Bundle-Version: 1.1.0.qualifier
```

Comment:
Again surprised this isn't requiring a major version bump because of the removed package.